target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-eap1300.dts

```diff
@@ -215,16 +219,31 @@
 	status = "okay";
 };
 
+&gmac {
+	status = "okay";
+};
+
+&switch {
+	status = "okay";
+};
+
+&swport5 {
+	status = "okay";
+	label = "lan";
+};
+
 &wifi0 {
 	status = "okay";
-	nvmem-cell-names = "pre-calibration";
-	nvmem-cells = <&precal_art_1000>;
+	nvmem-cell-names = "pre-calibration", "mac-address-ascii";
+	nvmem-cells = <&precal_art_1000>, <&macaddr_lan>;
+	mac-address-increment = <1>;
 	qcom,ath10k-calibration-variant = "EnGenius-EAP1300";
 };
 
 &wifi1 {
 	status = "okay";
-	nvmem-cell-names = "pre-calibration";
-	nvmem-cells = <&precal_art_5000>;
+	nvmem-cell-names = "pre-calibration", "mac-address-ascii";
+	nvmem-cells = <&precal_art_5000>, <&macaddr_lan>;
```

Comment:
And also here. However, this is just some naming scheme and not important for functionality.

Reply:
Please review my reply above and let me know if `macaddr_lan` still needs to be renamed.